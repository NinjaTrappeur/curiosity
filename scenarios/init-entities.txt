entity create one "One S.A." 100200300 BE0100200300
entity update one "One S.A. is the main legal entity of the prototype. Unless required for fiscal or legal reasons, new contracts are hosted in One."
entity link-user one USER-1 --validator
