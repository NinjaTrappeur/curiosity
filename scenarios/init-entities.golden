1: entity create one One S.A. 100200300 BE0100200300
Legal entity created: LENT-1
2: entity update one One S.A. is the main legal entity of the prototype. Unless required for fiscal or legal reasons, new contracts are hosted in One.
Legal entity updated: one
3: entity link-user one USER-1 --validator
Legal entity updated: one
